src/components/home/ProjectsBrief.jsx

```diff
@@ -1,8 +1,80 @@
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+// Custom Imports
+import projectsList from '../projects/projectsList';
+
+const cardStyle = {
+  display: 'flex',
+  // alignItems: 'center',
+  flexDirection: 'column',
+  mx: 2,
+  p: 4,
+  background: '#D9D9D9 40%',
+  borderRadius: '10px',
+  height: '100%',
+  textAlign: 'justify'
```

Comment:
`textAlign: 'justify'`
this just adds spacing that makes the text look weird.

Reply:
This isn't present in the latest version. You may be looking at an outdated version.